### DISCUSSION TITLE
can not start dbeaver

### DISCUSSION THREAD

**Discussion by ccczhouxin** [dbeaver/dbeaver#18130] [DBeaver Development]
### Description

I update the project still has error ,the eclipse is 2022.9
![image](https://user-images.githubusercontent.com/43168824/199935236-a69e6547-3a4b-447e-ba20-9265b3a44c1b.png)
![image](https://user-images.githubusercontent.com/43168824/199935307-13c7df6b-46d3-4cf2-8476-279856b7ca81.png)
![image](https://user-images.githubusercontent.com/43168824/199935355-5e511f3e-a6df-499d-ae08-343a953c2dcf.png)

### DBeaver Version

Community Edition 22.2.3

### Operating System

windows 64

**Comment by miltonchirinos44** [comment]
I changed the version of openjdk to 17 and it still doesn't work dbeaver, I still can't find a solution for this error

**Reply by ccczhouxin** [reply]
> I changed the version of openjdk to 17 and it still doesn't work dbeaver, I still can't find a solution for this error
Maybe only a few use newest dbeaver or cloudbeaver

**Comment by ccczhouxin** [comment]
> I changed the version of openjdk to 17 and it still doesn't work dbeaver, I still can't find a solution for this error
Yes，i don know the offical how to use successful

**Comment by miltonchirinos44** [comment]
Thanks for the help and I managed to solve the error

**Reply by ccczhouxin** [reply]
ok
